doc/deps.md

```diff
@@ -13,6 +13,7 @@ be installed from the official repository of the given Linux distribution.
 - **`cmake`** and **`make`**: For building CodeCompass.
 - **`g++`**: For compiling CodeCompass. A version which supports C++14 features
   is required. (Alternatively, you can compile with Clang.)
+- **`gcc-X`, `gcc-X-plugin-dev`**: For building ODB.
 - **`libboost-all-dev`**: Boost can be used during the development.
 - **`llvm-7-dev`**, **`libclang-7-dev`**: C++ parser uses LLVM/Clang for
```

Comment:
`.travis.yml` installs `clang-7` too. Unfortunately, this is needed so CMake's `FindClang` works...

```suggestion
- **`llvm-7-dev`**, **`clang-7`**, **`libclang-7-dev`**: C++ parser uses LLVM/Clang for
```